dp_creator_ii/tests/test_app.py

```diff
@@ -36,13 +36,13 @@ def expect_no_error():
 
     csv_path = Path(__file__).parent / "fixtures" / "fake.csv"
     page.get_by_label("Choose CSV file").set_input_files(csv_path.resolve())
-    expect_visible("student_id")
     expect_no_error()
 
     page.get_by_role("button", name="Define analysis").click()
     expect_not_visible(pick_dataset_text)
     expect_visible(perform_analysis_text)
     expect_not_visible(download_results_text)
+    expect_visible("student_id")
```

Comment:
the test could be updated to check for all columns in the data file

Reply:
done!